Tribler/community/doubleentry/database.py

```diff
@@ -0,0 +1,266 @@
+""" This file contains everything related to persistence for DoubleEntry.
+"""
+import base64
+
+from os import path
+
+from Tribler.dispersy.database import Database
+
+
+def encode_db(db_object):
```

Comment:
You're encoding a binary string into base64 and then storing it as a unicode string to get around the asserts of  dispersy regarding not accepting strings. Just use buffer object in the appropriate places. 

And explicitly convert some columns into buffer and not use a method like this.